### PR TITLE
Increment script L1->L2 now working in Cross Chain tutorial.

### PR DESCRIPTION
**Pull request by kailash-manasarovar**
- The script is fixed with added comments.
- Some other minor fixes.
- Running on latest Node.

### CODE REVIEW

**Comment by uF4No** [comment]
tested and added a few minor changes @kailash-manasarovar @krofax

File: docs/dev/tutorials/cross-chain-tutorial.md [hunk 1]
```diff
@@ -35,33 +35,7 @@ mkdir L1-governance L2-counter
...
-npm install --save-dev hardhat
-```
-@tab yarn
-```sh
-yarn add --dev hardhat
-```
-:::
-
-4. Run the following to set up the project:
+2. Run the following to initialise and set up the L1 project:
```
Comment (kailash-manasarovar) [inline docs/dev/tutorials/cross-chain-tutorial.md:38]: What if I've only just installed node?

Comment (kailash-manasarovar) [reply docs/dev/tutorials/cross-chain-tutorial.md:38]: I guess I have it from the quick start ...

**Review by uF4No — APPROVED** [review]
LGTM

**Pull request merged**